rholang/src/main/scala/coop/rchain/rholang/interpreter/RhoRuntime.scala

```diff
@@ -431,7 +431,8 @@ object RhoRuntime {
     "rho:registry:insertSigned:secp256k1" -> Bundle(
       FixedChannels.REG_INSERT_SIGNED,
       writeFlag = true
-    )
+    ),
+    "rho:registry:insertBlessed" -> Bundle(FixedChannels.REG_INSERT_BLESSED, writeFlag = true)
```

Comment:
Do we have test to check that anyone cannot call this process and insert arbitrary code?

Reply:
No tests for that. 
it is guarded by pattern match. https://github.com/rchain/rchain/blob/e166d539376041758a17add4d18519d683468fbb/casper/src/main/resources/Registry.rho#L584-L601

Reply:
I'd like to eliminate the need for `rho:registry:insertBlessed`. Can you please check if this is possible.

Reply:
Do you have any backup plans for replacing this? I tried and could not get a good solution for that.